### PR TITLE
Remove requirements/recommendations related to UI

### PR DESCRIPTION
**Pull request by subliun**

### CODE REVIEW

**Comment by ProMcTagonist** [comment]
Why?

**Comment by optimumtact** [comment]
The client should ensure that clients operate correctly with each other, not that they all look the same.

Let the commons choose which client's UX is the best.

In this case however these are probably fine as recommendations

**Comment by GrayHatter** [comment]
:+1:

**Comment by Chuongv** [comment]
:+1:

**Comment by ProMcTagonist** [comment]
@optimumtact 

> In this case however these are probably fine as recommendations

+1

**Comment by Zer0-One** [comment]
Alright, this looks overdue to merge. @GrayHatter @Chuongv @iphydf @JX7P @stal888 @JFreegman @tux3 @Impyy 

All in favor of merging, say :+1: , all opposed say :-1:

**Comment by Zer0-One** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -9,11 +9,6 @@ Introduction
   minimum, comply with every ![](badge/req.png) point in this document (except
   where otherwise stated).  For this reason, things that are ![](badge/req.png)
   should not be needlessly specific.
-    - For example, this is a **bad** ![](badge/req.png) point: *A Tox client
```
Comment (optimumtact) [inline README.md:12]: this is not valid to remove so :-1: until fixed

Comment (netbsduser) [inline README.md:12]: Can you explain what you mean by 'not valid to remove', please?

Comment (Zer0-One) [inline README.md:12]: yeah, i thought this was kind of cheesy, and sounded out-of-place

Comment (optimumtact) [inline README.md:12]: It's an example of how overly specific rules are bad.

Comment (Zer0-One) [inline README.md:12]: well sure, but i think you get this point across just by saying "overly specific rules are bad", which is what we've done here

Comment (optimumtact) [inline README.md:12]: ok :neutral_face:

**Comment by alexbakker** [comment]
:+1:

**Comment by netbsduser** [comment]
Looks good to me.

**Comment by GrayHatter** [comment]
:-1: the numbers need to be fixed

**Comment by Chuongv** [comment]
:+1:

**Comment by Zer0-One** [comment]
Votes to merge (need a majority):
- [ ] @grayhatter
- [ ] @Zer0-One 
- [x] @stal888  
- [ ] @JFreegman 
- [x] @JX7P 
- [ ] @tux3
- [ ] @iphydf
- [x] @Chuongv
- [x] @Impyy

**Comment by Zer0-One** [comment]
I'm afraid @GrayHatter is right, you can't just remove a section without fixing all of the section numbers after it. @subliun please make the appropriate corrections.

**Comment by GrayHatter** [comment]
That's right! you be afraid of my correctness!

**Comment by subliun** [comment]
These numbers are a pain. There should be a script to do this for me, or we should not worry about it.

I'd be willing to write one in scala.

**Comment by GrayHatter** [comment]
Write one in java, it'll be easy!!!

**Comment by subliun** [comment]
Numbers fixed, thus it gets my :+1:

**Comment by GrayHatter** [comment]
:+1: @Zer0-One

**Comment by stal888** [comment]
:mushroom:

**Comment by Zer0-One** [comment]
Alright, restarting the vote since the PR changed:

Votes to merge (need a majority):
- [x] @grayhatter
- [x] @Zer0-One 
- [x] @stal888  
- [ ] @JFreegman 
- [ ] @JX7P (no vote)
- [ ] @tux3 (no vote)
- [ ] @iphydf (no vote)
- [x] @Chuongv
- [x] @Impyy

**Comment by JFreegman** [comment]
:-1: I still disagree with removing the user status clause. Common sense should prevail over rigid adherence to rules when it benefits no one.

**Comment by Chuongv** [comment]
:+1:

**Comment by alexbakker** [comment]
:+1:

**Pull request merged**